### DISCUSSION TITLE
Alchemy账号邮箱无法注册的解决教程?

### DISCUSSION THREAD

**Discussion by wenki104** [openbuildxyz/solidity_bootcamp#4] [Q&A]
当我在用QQ\谷歌邮箱注册Alchemy账号时，出现了Blocked signup或者其他别的错误提示时，应该怎么解决？
![1700713204849](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/d70f7c9d-1994-4fc4-a8cf-4e59244118b0)

**Comment by wenki104** [comment]
# **Alchemy**

# **账号邮箱无法注册解决教程：**

**一：任何邮箱，在审核网站常规的提交资料走程序。（持续更新；待验证）**

使用方法：当我在用QQ邮箱注册时，出现了Blocked signup或者其他别的错误提示时，我都可以通过此方法规矩的提交验证信息来证明。

审核网站[https://alchemyapi.typeform.com/to/AScaKcSU](../inline)

下图是我的报错实例：

![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/23c12a48-a45f-48bf-b071-a6093719ee10)

按照流程提交信息说明

1. 
![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/ba055f90-6748-45b4-8427-290ddd1a5192)

2. 此处和一开始申请的名称一致即可 ![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/057b78de-3e70-47bc-b35f-abe0a59fdcb0)
3. 我是如实填写的 
![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/c49a1d0e-6f60-4f85-8b5c-41b44bc176fa)

4. 因为我用的QQ邮箱直接就是"Blocked signup"，所以我选择其他，然后如实填进去。 ![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/3a5c0597-ca26-4ec4-b740-cd8b42825f9b)
5. 我选的领水，因为也确实是领测试币。 然后我建议如实填写，因为后面要详细讲述原因 ![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/b58b8f18-8482-4ccf-b03d-ddb8c7623487)
6. Alchemy很强大。我说：我要学习网站上面关于账户抽象的技术课程，然后需要一些测试币来才能继续我的项目。 这两条很实在的原因，我建议大家写的过程中，加上一些区块链行业内的专业术语，最好具体一点，不要泛泛而谈，我这次就写到了关于钱包这个基础设施中，目前最火热的账户抽象技术。
翻译如下：I want to learn account abstraction technology and the latest areas about blockchain, such as RWA, through Alchemy. I really need some courses available on the website.And I nees some goerlifaucet ETH to continue my project .![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/476d3d1f-5907-4f9e-b899-4c02abeae45a)
7. ![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/a84d8371-db6f-4f22-aac4-de9f51c7856b)

**二：发邮件给官方邮件组。**

直接寻求邮箱进白名单的机会。 楼主也是通过这个方式，半天的时间就从被ban的谷歌邮箱，到注册登录都畅行无阻了。

收件邮件：[abuse@alchemy.com](../inline)

发件邮箱：我准备使用 **谷歌邮箱** 注册Alchemy账号，但被告知我的邮箱invaild（无效的）。

**申诉经历：**

1.

**行文思路：** 一定要真诚的表达需求。 我先是讲明来由：注册失败邮箱无效。 然后说我出我的困难：我只有这一个邮箱。 接着说出请求：将我的邮箱放入白名单里。

**我在邮件中是这样写的：**

我差一步就注册成功了，但网页显示我的邮箱无效。我没有别的邮箱了，你能告诉我怎么才能注册成功吗？请求官方直接将我的邮箱放到白名单里，或者给我一些测试币直接打到我的地址里来。英语不是我的母语，所以可能存在语法错误，请见谅。

**这样写的好处：**

逻辑清晰：为了领水所以才注册。 因为没有别的邮箱，所以才向官方邮件组提出请求。显得真实可信。 最后的要求也十分直接：拉到白名单里，一切都能解决。这里我还说不拉进白名单也可以，直接给我打水到链上，这个同样呼应了我是一个真实领水的测试用户。加强了可信度。大家可以用一个简单具体的理由前后呼应，自圆其说就可以。一定要真诚、语言不要死板，配合chatGPT可完美解决。

**原版邮件：**

I am just one step away from receiving it, but it reminds me that this email address is not valid. I don't have any other email address. Can you tell me how to register successfully?Or you can help me put this email address into the white name list or give me the eth address to directly issue test coins. It's up to you. I'm not British, so you may see some grammatical mistakes, please forgive me. ![](RackMultipart20231123-1-oh7eft_html_39f51ec5c427a43.png)

Address:

Email:

![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/664e7056-2d25-497b-bf8f-e85e370f9e18)

2. 第二天，当我再次申请时，就显示此邮箱已经完成注册了。这里，我的邮箱已经是被拉到白名单里了。

翻译：

"您已经拥有一个Alchemy账户

您好 Awhook，

您使用邮箱 dieseld848@gmail.com 请求创建一个账户，但是使用该邮箱地址的账户已经存在。

如果您记得密码，您可以立即登录。否则，您可以重置密码。" ![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/89662404-426e-4a13-8735-68d601db3905)

3. 就按照邮件链接重置密码就可以了，因为昨天根本就是无效的，不存在注册成功一说。

这是我重置密码的邮件 ![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/5d5dde3a-be52-49f5-a9e8-3d3f625a5311)

4. 这就登录成功了。

此封邮件的意思是：欢迎您加入Alchemy平台的邮件。它提供了一系列资源和链接，帮助您开始使用Alchemy构建Web3应用程序。这些资源包括开发者中心、API文档、教程、YouTube系列视频、社区支持渠道（如Discord）、教程中心和Twitter，旨在帮助您提升Web3开发技能并了解Alchemy平台的最新动态。邮件的最后提到了Nikil、Joe和Alchemy团队，表示对您开始这段旅程的期待和支持。![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/bd386890-97dd-4352-819c-b7443bf53d4a)

5. ![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/3a31ce90-8514-4a87-98e8-29d57159e412)
![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/0e87590d-17c2-416e-aad0-bbac2e4a26d7)
![image](https://github.com/openbuildxyz/solidity_bootcamp/assets/140960805/df9db236-bc4a-46e4-8ab4-0e346990e5d4)


我第一次发邮件的时间是当天15：42。将我邮箱拉入白名单的时间是在12：41之前发生的。因为那个时间，我才发现我邮箱从无效的状态变成为已注册状态。之后重置密码加登陆就很快完成了。 希望我的这种Alchemy注册经历，也能够给有需要的人提供一种解决思路。

**Comment by 201917316** [comment]
不行了，我申请不回复我

**Reply by chenchen53698** [reply]
现在呢

**Comment by fanofmusk** [comment]
您的邮件已收到，谢谢
